src/components/index.jsx

```diff
@@ -32,33 +34,68 @@ function Home() {
 	useEffect(() => {
 		client.get(baseURL).then(setJobs);
 	}, []);
+	const [open, setOpen] = useState(false);
+
 	return (
 		<>
 			<div className="container">
 				<div className="main-body">
 					<h2>Search For Developer Jobs</h2>
 					<div className="main-card">
-						<div>
+						<div className={cn(' form-wrapper', open && 'active')}>
 							<Form className="search-part">
-								<Form.Group>
-									<Form.Label className="form-label">Description</Form.Label>
-									<Form.Control placeholder="Javascript,Python etc" type="text" name="description" className="custom-text" />
-								</Form.Group>
-
 								<Form.Group>
 									<Form.Label className="form-label">Location</Form.Label>
 									<Form.Control placeholder="Where?" type="text" name="location" className="custom-text" />
 								</Form.Group>
-
 								<Form.Group>
 									<Form.Label className="form-label">Type</Form.Label>
-									<Form.Control as="select" custom className="custom-text">
-										<option>Full Time</option>
-										<option>Part Time</option>
-									</Form.Control>
+									<div className="custom-text text-center">
```

Comment:
`custom-text` is too generic of a class name

Reply:
changed it to custom-checkbox